### DISCUSSION TITLE
zh/guide/drivers/feiji

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#324] [Announcements]
# zh/guide/drivers/feiji

小飞机网盘：https://feijipan.com/ 根文件夹ID 根目录ID，默认为0，其它目录ID查看下图获取方式 账户、密码 填写自己的小飞机网盘帐号密码 默认使用的下载方式

https://alist.nn.ci/zh/guide/drivers/feiji.html

**Comment by a49942245a** [comment]
v3.30  添加储存找不到小飞机网盘的驱动

**Reply by 8MiYile** [reply]
在下个版本才支持，不知道为啥就提取发文章了

**Reply by 98654lemon** [reply]
更新吧，过年啦

**Comment by shenyuanheli** [comment]
升级了 3.3 版本 怎么没有 小飞机 网盘的驱动啊？

**Comment by ghx0xp** [comment]
3.31的能上传但是下载不了啊，提示{"msg":"参数有误","code":-1}，怎么办？

**Reply by pda8888** [reply]
我也是：
3.31的能上传但是下载不了啊，提示{"msg":"参数有误","code":-1}，怎么办？

**Comment by huolanyun** [comment]
linux可以自己编译，替换最新

**Comment by Jerry-Z07** [comment]
现在3.31，能上传，但下载不了，报failed link: failed get link: redirect failed, status: 200

**Reply by 8MiYile** [reply]
v3.33也不行

**Reply by vpeu** [reply]
failed link: failed get link: redirect failed, status: 200

**Comment by nellios** [comment]
failed link: failed get link: redirect failed, status: 200

**Reply by arch3rPro** [reply]
failed link: failed get link: redirect failed, status: 200

**Comment by AoGou3175** [comment]
同问，3.34    能挂载无法下载，wevdav上传无限套娃。。。

**Comment by vpei** [comment]
小公司出品的不稳定，也不靠谱。🤢

**Comment by suntao12138** [comment]
小飞机，不能下载，不能上传，貌似网盘那边做了更改

**Comment by Yurisu** [comment]
版本 : v1.1.1更新时间 : 2024-06-04
小飞机网盘可以挂载，正确显示文件列表、文件名称、文件大小、修改时间，可以创建文件夹。
无法重命名文件夹，无法下载，无法上传，新建文件夹后里面的内容显示为根目录下内容（但实际网盘是没有文件的）

**Comment by vpei** [comment]
文件夹文件一多，Alist就打不开文件夹，这个大家可遇到过？
